### PR TITLE
ADEN-1818 Extend the krux tests to mobile.

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 1]
```diff
@@ -115,6 +115,11 @@ private String updateUrl(String page) {
     return page;
   }
 
+  public void getUrl(String wikiName, String article) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: I think a better place for dealing with URLs is urlBuilder

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 2]
```diff
@@ -696,23 +701,19 @@ public void verifyGptAdInSlot(String slotName, String src, String lineItemId, St
     );
   }
 
-  protected boolean isGptParamPresent(String key, String value) {
-    waitForElementByElement(presentMedrec);
-    String dataGptPageParams =
-        presentLeaderboardGpt.getAttribute("data-gpt-page-params").replaceAll("[\\[\\]]", "");
+  protected boolean isGptParamPresent(WebElement slot, String key, String value) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: Enoum would be nice for a slot

File: src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java [hunk 3]
```diff
@@ -16,12 +22,22 @@ public UrlBuilder() {
...
   public String getUrlForPath(String wikiName, String wikiPath) {
     String url = getUrlForWiki(wikiName);
     if (!(isWikia)) {
       url += "wiki/";
     }
     url += wikiPath;
+    return updateUrl(url);
+  }
+
+  private String updateUrl(String url) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java]: You could use more descriptive name

Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java]: And JavaDoc would be nice

**Comment by gabrys** [comment]
LGTM

**Pull request merged**